jsonparser/grafana.go

```diff
@@ -0,0 +1,219 @@
+package jsonparser
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/rudderlabs/rudder-go-kit/jsonrs"
+
+	"github.com/grafana/jsonparser"
+)
+
+// grafanaJSONParser is the implementation of JSONParser using jsonparser library
+type grafanaJSONParser struct{}
+
+// GetValue retrieves the value for a given key from JSON bytes using jsonparser
+func (p *grafanaJSONParser) GetValue(data []byte, keys ...string) (interface{}, error) {
```

Comment:
[nitpick] Can we use `any` instead of `interface{}` as it's clearer? Same everywhere else in this PR.